### ISSUE TITLE
Running Jaeger-operator in kubernetes - URL not working - help required

### ISSUE THREAD

**Issue by get2arun** [jaegertracing/jaeger-operator#528]
Hi Team,
I have followed the jager-operator tutorial for Kubernetes to run jaeger in my local Ubuntu laptop. All the commands are executed without any errors. However, I could not access the URL and it is not reachable.

Do I need any configuration or setup to bring the Jaeger-Operator URL running. 
Do I need to run Ingress or nginx to make the URL available external. 
I am new to Kubernetes, all I need is to access Jaeger URL accessible in Browser.

```
root@arun-desktop:/home/data-common/jaeger# kubectl get deployment jaeger-operator -n observability
NAME              READY   UP-TO-DATE   AVAILABLE   AGE
jaeger-operator   1/1     1            1           16s

```
```
root@arun-desktop:/home/data-common/jaeger# kubectl get jaeger
NAME       AGE
simplest   11m
```

```
root@arun-desktop:/home/data-common/jaeger# kubectl get service
NAME                          TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)                                  AGE
kubernetes                    ClusterIP   10.96.0.1        <none>        443/TCP                                  22m
simplest-agent                ClusterIP   None             <none>        5775/TCP,5778/TCP,6831/TCP,6832/TCP      11m
simplest-collector            ClusterIP   10.101.169.178   <none>        9411/TCP,14250/TCP,14267/TCP,14268/TCP   11m
simplest-collector-headless   ClusterIP   None             <none>        9411/TCP,14250/TCP,14267/TCP,14268/TCP   11m
simplest-query                ClusterIP   10.103.196.1     <none>        16686/TCP                                11m

root@arun-desktop:/home/data-common/jaeger# kubectl get pods
NAME                        READY   STATUS    RESTARTS   AGE
simplest-56c7bd47bf-z2b9r   1/1     Running   0          12m

root@arun-desktop:/home/data-common/jaeger# kubectl get daemonsets
No resources found.

root@arun-desktop:/home/data-common/jaeger# kubectl get ingress
NAME             HOSTS   ADDRESS     PORTS   AGE
simplest-query   *       10.0.2.15   80      12m
```

```
root@arun-desktop:/home/data-common/jaeger# kubectl logs simplest-56c7bd47bf-z2b9r
2019/07/15 15:58:19 maxprocs: Leaving GOMAXPROCS=2: CPU quota undefined
{"level":"info","ts":1563206299.9074147,"caller":"flags/service.go:115","msg":"Mounting metrics handler on admin server","route":"/metrics"}
{"level":"info","ts":1563206299.9085727,"caller":"flags/admin.go:108","msg":"Mounting health check on admin server","route":"/"}
{"level":"info","ts":1563206299.9090533,"caller":"flags/admin.go:114","msg":"Starting admin HTTP server","http-port":14269}
{"level":"info","ts":1563206299.9096763,"caller":"flags/admin.go:100","msg":"Admin server started","http-port":14269,"health-status":"unavailable"}
{"level":"info","ts":1563206299.9815555,"caller":"memory/factory.go:55","msg":"Memory storage initialized","configuration":{"MaxTraces":0}}
{"level":"info","ts":1563206300.0397136,"caller":"grpc/builder.go:75","msg":"Agent requested insecure grpc connection to collector(s)"}
{"level":"info","ts":1563206300.0410292,"caller":"grpc/clientconn.go:242","msg":"parsed scheme: \"\"","system":"grpc","grpc_log":true}
{"level":"info","ts":1563206300.0434556,"caller":"grpc/clientconn.go:248","msg":"scheme \"\" not registered, fallback to default scheme","system":"grpc","grpc_log":true}
{"level":"info","ts":1563206300.0435498,"caller":"grpc/resolver_conn_wrapper.go:113","msg":"ccResolverWrapper: sending update to cc: {[{127.0.0.1:14250 0  <nil>}] }","system":"grpc","grpc_log":true}
{"level":"info","ts":1563206300.0447922,"caller":"base/balancer.go:76","msg":"base.baseBalancer: got new resolver state: {[{127.0.0.1:14250 0  <nil>}] }","system":"grpc","grpc_log":true}
{"level":"info","ts":1563206300.0455194,"caller":"base/balancer.go:130","msg":"base.baseBalancer: handle SubConn state change: 0xc0004eaa10, CONNECTING","system":"grpc","grpc_log":true}
{"level":"info","ts":1563206300.0473614,"caller":"grpc/clientconn.go:1139","msg":"grpc: addrConn.createTransport failed to connect to {127.0.0.1:14250 0  <nil>}. Err :connection error: desc = \"transport: Error while dialing dial tcp 127.0.0.1:14250: connect: connection refused\". Reconnecting...","system":"grpc","grpc_log":true}
{"level":"info","ts":1563206300.0476372,"caller":"base/balancer.go:130","msg":"base.baseBalancer: handle SubConn state change: 0xc0004eaa10, TRANSIENT_FAILURE","system":"grpc","grpc_log":true}
{"level":"info","ts":1563206300.048041,"caller":"all-in-one/main.go:198","msg":"Starting agent"}
{"level":"info","ts":1563206300.0493066,"caller":"app/agent.go:68","msg":"Starting jaeger-agent HTTP server","http-port":5778}
{"level":"info","ts":1563206300.080141,"caller":"all-in-one/main.go:241","msg":"Starting jaeger-collector TChannel server","port":14267}
{"level":"info","ts":1563206300.0815785,"caller":"grpcserver/grpc_server.go:64","msg":"Starting jaeger-collector gRPC server","grpc-port":"14250"}
{"level":"info","ts":1563206300.0820014,"caller":"all-in-one/main.go:259","msg":"Starting jaeger-collector HTTP server","http-port":14268}
{"level":"info","ts":1563206300.0825968,"caller":"querysvc/query_service.go:130","msg":"Archive storage not created","reason":"archive storage not supported"}
{"level":"info","ts":1563206300.0835032,"caller":"all-in-one/main.go:341","msg":"Archive storage not initialized"}
{"level":"info","ts":1563206300.0864298,"caller":"healthcheck/handler.go:129","msg":"Health Check state change","status":"ready"}
{"level":"info","ts":1563206300.0865753,"caller":"app/server.go:134","msg":"Starting CMUX server","port":16686}
{"level":"info","ts":1563206300.0873005,"caller":"app/server.go:111","msg":"Starting HTTP server","port":16686}
{"level":"info","ts":1563206300.0874696,"caller":"app/server.go:124","msg":"Starting GRPC server","port":16686}
{"level":"info","ts":1563206300.1425006,"caller":"all-in-one/main.go:297","msg":"Listening for Zipkin HTTP traffic","zipkin.http-port":9411}

```
What am I missing here.

**Comment by jpkrohling** [comment]
As per your `kubectl get ingress`, your Jaeger instance's UI should be available at http://10.0.2.15. For development purposes, you might also want to check out `kubectl port-forward`:

```console
$ kubectl get pods
NAME                        READY     STATUS    RESTARTS   AGE
simplest-5d89c8f6bf-nlzf8   1/1       Running   0          26s
$ kubectl port-forward simplest-5d89c8f6bf-nlzf8 16686:16686
Forwarding from 127.0.0.1:16686 -> 16686
Forwarding from [::1]:16686 -> 16686
$ firefox http://localhost:16686
```

**Comment by get2arun** [comment]
@jpkrohling - Thanks for information. It worked now.
